src/frontend/next/src/components/MyFeedsPage/HelpPopoverButton.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, IconButton, Popover, Typography } from '@material-ui/core';
```

Comment:
I don't think we should have a `MyFeedsPage` directory.  With next.js, we have directory for pages, and another for components.  The myfeeds page can be separated out.  Let's move this directly into the `components/` dir.

Reply:
Hi @humphd, thank you for your feedback! I've removed `MyFeedsPage` and saved the `HelpPopoverButton` into `components/` dir.